vkcs/db_config_group.go

```diff
@@ -0,0 +1,56 @@
+package vkcs
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func extractDatabaseConfigGroupValues(rawValues map[string]interface{}, dsParameterTypes map[string]string) (map[string]interface{}, error) {
+	values := make(map[string]interface{})
+
+	for name, value := range rawValues {
+		if vType, ok := dsParameterTypes[name]; ok {
+			switch vType {
+			case "string":
+				values[name] = value
+			case "integer":
+				if valueInt, err := strconv.Atoi(value.(string)); err == nil {
+					values[name] = valueInt
+				} else {
+					return nil, fmt.Errorf("incorrect value type")
+				}
+			case "boolean":
+				if valueBool, err := strconv.ParseBool(value.(string)); err == nil {
+					values[name] = valueBool
+				} else {
+					return nil, fmt.Errorf("incorrect value type")
```

Comment:
how do we let user know what parameter has incorrect type?

Reply:
Expanded error message